sqlglot/parser.py

```diff
@@ -1186,6 +1191,24 @@ def _parse_unnest(self):
             alias=alias,
         )
 
+    def _parse_derived_table_values(self):
+        is_derived = self._curr is not None and self._curr.token_type == TokenType.L_PAREN
+        if not self._match(TokenType.VALUES) and not self._match_pair(TokenType.L_PAREN, TokenType.VALUES):
+            return None
+
+        expressions = self._parse_csv(self._parse_paren_enclosed_csv, self._parse_column)
```

Comment:
i don't think any of these new methods are needed, just do

self._parse_value()

Reply:
Damn thanks